install/cluster-operator/050-ConfigMap-strimzi-cluster-operator.yaml

```diff
@@ -14,7 +14,12 @@ data:
     appender.console.layout.type = PatternLayout
     appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 
-    rootLogger.level = ${env:STRIMZI_LOG_LEVEL:-INFO}
+    appender.console.filter.filter1.type = MarkerFilter
+    appender.console.filter.filter1.onMatch = ACCEPT
+    appender.console.filter.filter1.onMismatch = DENY
+    appender.console.filter.filter1.marker = Kafka(myproject/my-cluster)
```

Comment:
Should this be commented out as an example? If yes, where do I set the log level for this marker?

Reply:
Yes. Maybe it should be moved to the doc. The marker is something like a filter. The level is still set on the logger. The marker is then just picking the logging messages matching the marker.

Reply:
I'm confused ... I thought that this allows you to for example set DEBUG log level for marker `Kafka(myproject/my-cluster)` and keep INFO log level for the rest. As you now explained it, it sounds more like all you can do is to switch off logging for all other markers than `Kafka(myproject/my-cluster)` - which would be a lot less useful I guess.